recipes/libusb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libusb" %}
+{% set version = "1.0.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 'https://github.com/libusb/libusb/releases/download/v{{ version}}/{{ name }}-{{ version }}.tar.bz2'
+  sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libusb') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('libudev-devel') }}  # [linux]
+    - libtool  # [unix]
+  host:
+
+test:
+  commands:
+    - test -f $PREFIX/include/libusb-1.0/libusb.h   # [unix]
+    - test -f $PREFIX/lib/libusb-1.0.a              # [unix]
+    - test -f $PREFIX/lib/libusb-1.0${SHLIB_EXT}    # [unix]
+    - conda inspect linkages -p $PREFIX libusb      # [unix]
+    - conda inspect objects -p $PREFIX libusb       # [osx]
```

Comment:
I think the `conda inspect` commands are deprecated. `conda-build` also provides the same info at the end of the build. Though this can be handled in the feedstock.